internal/registry/types.go

```diff
@@ -0,0 +1,26 @@
+package registry
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	DockerRegistryGVR = schema.GroupVersionResource{
```

Comment:
Now we have file `internal/btp/operator/gvr.go`. Isn't it good place for this definition?

Reply:
internal/btp package is related to the `btp` platform. And all sub-directories should be have the same destiny. Of course I can move these lines into the `internal/registry/gvr.go` file, but I think `types.go` file is generic enough